src/Encoding.ts

```diff
@@ -1,31 +1,31 @@
-'use strict';
```

Comment:
Added tslint. We can add more rules later. I already found a few bugs but don't want to refactor right now.